src/components/molecules/RingCharts/RingCharts.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import CustomRingChart from "../../atoms/CustomRingChart/CustomRingChart";
```

Comment:
si tenes ganas, solo si tenes ganas, podemos ver de usar alias aca y que quede algo asi "@atoms/CustomRingChart/CustomRingChart";

Reply:
https://nimblewebdeveloper.com/blog/absolute-alias-imports-in-javascript-vscode